django_redis/util.py

```diff
@@ -8,7 +8,7 @@
 
 
 @python_2_unicode_compatible
-class CacheKey(str):
+class CacheKey(object):
```

Comment:
Even though `CacheKey` is no longer a string, it still has a `__str__` method (or `__unicode__` on Python 2). When we use an instance of this class as an argument for a Redis command, this method will convert it to the contained cache key. And why do you need to call `len()` or `format` on a cache key wrapper?